### PR TITLE
docs: Add more presentable README

### PR DESCRIPTION
**Pull request by samir-a-ts**
## Related tasks

#7 

## Dependence on other Pull Requests (optional)

#7

## Changes

Added more presentable README for yandex_mapkit_lite package.

Screenshot 2024-02-08 at 23 22 51

## Troubles (optional)

There is a lot of work to be done on this package - not only the examples and their appearence, but also some bugs should be fixed ASAP.

## Checklist for self-check
- [x] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [x] The author is marked as an assigne and assigned mandatory reviewers.
- [x] Required labels marked
- [x] Specified related tasks and/or related PRs.
- [x] Specified Changes.
- [x] Attached videos/screenshots demonstrating the fix/feature.
- [x] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

**Review by internetova — COMMENTED** [review]
Thank you so much for such a beautiful design! 
To complete the automation, you need to add a Description to README.md

**Review by internetova — APPROVED** [review]
🔥

File: README.md [hunk 1]
```diff
@@ -1,71 +1,59 @@
-# yandex_mapkit_lite
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/b1ca707b-b162-43d5-a9b1-d1895ab99e14">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/f88243c6-26f0-4491-93bd-baf4ad5a5e50">
+  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/f88243c6-26f0-4491-93bd-baf4ad5a5e50">
+</picture>
 
 [![Build Status](https://shields.io/github/actions/workflow/status/surfstudio/yandex-mapkit-lite-flutter/main.yml?logo=github&logoColor=white)](https://github.com/surfstudio/yandex-mapkit-lite-flutter)
 [![Coverage Status](https://img.shields.io/codecov/c/github/surfstudio/yandex-mapkit-lite-flutter?logo=codecov&logoColor=white)](https://app.codecov.io/gh/surfstudio/yandex-mapkit-lite-flutter)
```
Comment (maxim-sysoev) [inline README.md:8]: guess needs to remove coverage, there are no tests

File: README.md [hunk 2]
```diff
@@ -1,71 +1,59 @@
...
 [![Pub Likes](https://badgen.net/pub/likes/yandex_mapkit_lite)](https://pub.dev/packages/yandex_mapkit_lite)
 [![Pub popularity](https://badgen.net/pub/popularity/yandex_mapkit_lite)](https://pub.dev/packages/yandex_mapkit_lite/score)
 ![Flutter Platform](https://badgen.net/pub/flutter-platform/yandex_mapkit_lite)
+[![License: MIT](https://img.shields.io/badge/license-MIT-purple.svg)](https://opensource.org/licenses/MIT)
 
-## Description
+---------
 
-**This library is a fork of the package yandex_mapkit 3.4.0.**
+Fork of [yandex_mapkit](https://pub.dev/packages/yandex_mapkit) library, but less heavy and more optimized.
```
Comment (maxim-sysoev) [inline README.md:17]: > more optimized

what optimization work has been done?

File: README.md [hunk 3]
```diff
@@ -1,71 +1,59 @@
...
+Made by [Surf :surfer:](https://surf.dev/flutter/) Flutter team :cow2:
 
-**yandex_mapkit_lite** is based on MapKit lite version.
+## Description
 
-Using the lite version of MapKit allows you to reduce the application size by an average of 30%.
+- :earth_africa: Map overview - enables to view the map of the world, with which user can interact with any convinient way, usually in order to demonstrate the location of some place
+- :house: Custom map objects - enables for developers to add custom map objects in order to indicate some place on the map
+- :video_game: Convinient map controls - there is an API for straight-to-point map controls through the code - from zooming and moving to limiting user scroll and controlling the speed
+- :leaves: App bundle size reduction - the average bundle size reduction for 25% comparing to projects with [original package](https://pub.dev/packages/yandex_mapkit)
```
Comment (maxim-sysoev) [inline README.md:26]: > bundle size reduction for 25% comparing to projects with original package

Does the size of the map increase proportionally to the size of the application?

File: .markdownlint.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (maxim-sysoev) [inline .markdownlint.json:1]: is this file required?

File: README.md (hunk 1 above)
Comment (internetova) [reply README.md]: ✅

File: README.md (hunk 2 above)
Comment (samir-a-ts) [reply README.md:16]: So, here is the backstory:

There were some additions to version of the package on the project where I am, that we added for "optimization" or so we thought, however after extensive research I found out that there are no changes from our so-called "optimization". 

So I added this "more optimized" thing just in advance, because in fact I had plans for transfering "optimized" code here. I agree, I will paraphrase it.

File: README.md (hunk 3 above)
Comment (samir-a-ts) [reply README.md:25]: I believe that no. I will think how to paraphrase it.

File: .markdownlint.json (hunk 4 above)
Comment (samir-a-ts) [reply .markdownlint.json:1]: Just to get rid of warnings when fixing readme files - it will "warn" you about the inline HTML and other things

File: README.md (hunk 2 above)
Comment (samir-a-ts) [reply README.md:16]: There is no optimization required, however users should be aware of some features of native maps implementation. They all are described in new example project.

Comment (samir-a-ts) [reply README.md]: ✅

File: README.md (hunk 3 above)
Comment (samir-a-ts) [reply README.md]: ✅

**Review by maxim-sysoev — APPROVED** [review]

**Review by mpkander — APPROVED** [review]
